### PR TITLE
add real data to buildings fragment

### PR DESCRIPTION
**Pull request by rekagiday**

### CODE REVIEW

**Comment by adamgyulavari** [comment]
@rekagiday conflicts

File: app/src/main/java/com/greenfox/tribesoflagopusandroid/fragments/BuildingsFragment.java [hunk 1]
```diff
@@ -9,43 +9,56 @@
...
-        Building farm1 = new Building(2, "farm", 1, 2);
-        Building farm2 = new Building(3, "farm", 1, 2);
-        Building mine1 = new Building(4, "mine", 1, 2);
-        Building mine2 = new Building(5, "mine", 1, 2);
-        Building barrack1 = new Building(6, "barrack", 1, 5);
-        Building barrack2 = new Building(7, "barrack", 1, 5);
+            }
+        });
 
         buildingsAdapter = new BuildingsAdapter(this.getContext(), buildings);
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/tribesoflagopusandroid/fragments/BuildingsFragment.java]: @rekagiday ask @viktorhegyi :D

**Review by adamgyulavari — DISMISSED** [review]

File: app/src/main/java/com/greenfox/tribesoflagopusandroid/fragments/BuildingsFragment.java (hunk 1 above)
Comment (rekagiday) [reply app/src/main/java/com/greenfox/tribesoflagopusandroid/fragments/BuildingsFragment.java]: removed .this and checked, it works fine, doesn't throw nullpointer

Comment (adamgyulavari) [reply app/src/main/java/com/greenfox/tribesoflagopusandroid/fragments/BuildingsFragment.java]: ask again :)

**Review by viktorhegyi — APPROVED** [review]

**Pull request merged**